crypto/bn/rsaz_exp_x2.c

```diff
@@ -575,12 +576,6 @@ static void to_words52(BN_ULONG *out, int out_len,
         out++;
         out_len--;
     }
-
-    while (out_len > 0) {
-        *out = 0;
-        out_len--;
-        out++;
-    }
```

Comment:
Would it be better to do here:
```
    if (out_len > 0)
        memset(out, 0, out_len * sizeof(BN_ULONG));
```

Reply:
yes, and the if should also be unnecessary, since out_len must be >= 0 here.